Loop/Managers/DeviceDataManager.swift

```diff
@@ -102,6 +113,41 @@ final class DeviceDataManager {
         setupPump()
         setupCGM()
     }
+
+    var isCGMManagerValidPumpManager: Bool {
```

Comment:
Not a PR comment, just to help me understand. A CGM Manager can also be a valid Pump Manager?

Reply:
A class can implement both protocols, like MinimedPumpManager does (the pump is the main receiver of BG data from Enlite sensors)